diff --git a/Cargo.toml b/Cargo.toml
index 1357b34..b9f79f9 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,6 +36,6 @@ tokio        = {version = "1.6", features = ["io-std", "macros", "rt", "test-uti
 toml         = "0.5"
 tower-lsp    = "0.17"
 
-#[patch.crates-io]
-#sv-parser = {path = "../sv-parser/sv-parser"}
-#svlint    = {path = "../svlint"}
+[patch.crates-io]
+sv-parser = {path = "../sv-parser/sv-parser"}
+svlint    = {path = "../svlint"}
